app/build.gradle

```diff
@@ -75,17 +75,35 @@ dependencies {
 
     compile fileTree(dir: 'libs', include: ['*.jar'])
 
+    compile 'javax.annotation:jsr250-api:1.0'
+    compile 'com.squareup.okhttp3:okhttp-android-support:3.2.0'
+    compile 'com.squareup.retrofit2:retrofit:2.0.0'
+    compile 'com.squareup.retrofit2:adapter-rxjava:2.0.0'
+    compile 'com.squareup.retrofit2:converter-gson:2.0.0'
     compile 'com.github.bumptech.glide:glide:3.7.0'
     compile 'com.android.support:recyclerview-v7:23.3.0'
 
-    // Schematic
-    apt 'net.simonvt.schematic:schematic-compiler:0.6.7'
-    compile 'net.simonvt.schematic:schematic:0.6.7'
+    // RxJava
+    compile 'io.reactivex:rxjava:1.1.3'
+    compile 'io.reactivex:rxandroid:1.1.0'
```

Comment:
This file should be left unchanged. These libraries are already included in appshared/build.gradle, so this would just duplicate the dependency declarations